MaxText/layers/quantizations.py

```diff
@@ -222,13 +244,11 @@ def _get_quant_config(config):
     return None
   if config.quantization == "int8":
     return _get_int8_quant_config(config)
-  if config.quantization == "int8w":
-    return _get_weight_only_quant_config(lhs_bits=None, rhs_bits=8)
-  if config.quantization == "int4w":
-    return _get_weight_only_quant_config(lhs_bits=None, rhs_bits=4)
   if config.quantization == "intmp":
```

Comment:
Why are you removing support for int/8int4 weights only?

Reply:
Ooh wait, are you expecting these to be added through MP configs? In that case can you also add a config for int4 weights only similar to int8 weight only config?

Reply:
Also need to update the comments here -- https://github.com/AI-Hypercomputer/maxtext/blob/main/MaxText/configs/base.yml#L74-L75